### DISCUSSION TITLE
Support for Uniformer Pose Estimation 支持 Uniformer Pose Estimation

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#310] [MMPose]
# Support for Uniformer Pose Estimation 
 支持 Uniformer Pose Estimation
 

*Migrate [https://github.com/Sense-X/UniFormer/tree/main/pose_estimation](https://github.com/Sense-X/UniFormer/tree/main/pose_estimation)  to the MMPose-1.0 architecture, ensuring alignment of inference accuracy with the open-source checkpoints. You can choose to migrate it to the projects/ directory (without writing unit tests) or to the mmpose/ directory (with the requirement of writing unit tests).*

将 [https://github.com/Sense-X/UniFormer/tree/main/pose_estimation](https://github.com/Sense-X/UniFormer/tree/main/pose_estimation)  迁移到 MMPose-1.0 架构，完成开源ckpt的推理精度对齐。可以自行选择迁入projects/下（不用写单元测试），或迁入mmpose/下（需要写单元测试）

## Task Information

**Repository**：MMPose

**Difficulty**：Hard

**Points**：50

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：xin-li-67

**Claim Time**：2023年06月14日

**PR Link**：https://github.com/open-mmlab/mmpose/pull/2501

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Tau-J** [comment]
This PR has been merged in https://github.com/open-mmlab/mmpose/pull/2501

**Reply by crazysteeaam** [reply]
got it~
